todo/models.py

```diff
@@ -1,6 +1,25 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from django.contrib.auth.models import User
 from django.db import models
 
+
 # Create your models here.
+
+
+class Task(models.Model):
+    TODO = "todo"
+    IN_PROGRESS = "in_progress"
+    DONE = "done"
+    TO_DELETE = "to_delete"
+
+    STATUS_CHOICES = [(TODO, "todo"), (IN_PROGRESS, "in_progress"), (DONE, "done"), (TO_DELETE, "to_delete")]
```

Comment:
1. tuple is better
2. Second parameter is human-readable value used in frontend, what is the idea of having it like that: 
 ("in_progress", "in_progress") ?
`STATUS_CHOICES = (
    (TODO, "Todo"), 
    (IN_PROGRESS, "In Progress"), 
    (DONE, "Done"), 
    (TO_DELETE, "To delete")]
`